meta/client.go

```diff
@@ -0,0 +1,92 @@
+package meta
+
+import (
+	"time"
+
+	"github.com/gogo/protobuf/proto"
+	"github.com/influxdb/influxdb/influxql"
+	"github.com/influxdb/influxdb/meta/internal"
+)
+
+type Client struct{}
+
+// CreateContinuousQuery creates a new continuous query.
+func (c *Client) CreateContinuousQuery(q *influxql.CreateContinuousQueryStatement) error
+
+// DropContinuousQuery removes a continuous query.
+func (c *Client) DropContinuousQuery(q *influxql.DropContinuousQueryStatement) error
+
+// CreateNode adds a new node to the cluster.
+func (c *Client) CreateNode(host string) error {
+	return c.exec(
+		internal.Command_CreateNodeCommand,
+		internal.E_CreateNodeCommand_Command,
+		&internal.CreateNodeCommand{
+			Host: proto.String(host),
+		})
+}
```

Comment:
This is an example of a client function. It simply wraps the protobuf encoding and passes it to the server.